packages/framework/src/client/__tests__/index.spec.js

```diff
@@ -37,10 +37,39 @@ describe( 'ApiClient', () => {
 	};
 
 	it( 'should initialize to empty state', () => {
-		const apiClient = new ApiClient( emptyApiSpec );
+		const apiClient = new ApiClient( emptyApiSpec, null );
 		expect( apiClient.state ).toEqual( {} );
 	} );
 
+	it( 'should stop scheduler when hidden and update scheduler when visible', () => {
+		let eventListener = null;
+		const doc = {
+			addEventListener: ( name, listener ) => {
+				eventListener = listener;
+			},
+			visibilityState: 'visible',
+		};
+		const apiClient = new ApiClient( emptyApiSpec, doc );
+		apiClient.scheduler = {
+			stop: jest.fn(),
+			updateDelay: jest.fn(),
+		};
+
+		expect( eventListener ).not.toBe( null );
+
+		// Simulate tab hidden
+		doc.visibilityState = 'hidden';
+		eventListener()
```

Comment:
One of the missing semi-colons reported by linting (I think this also caused the Travis build to fail?).